### ISSUE TITLE
ExifInterface.getExifDateTaken supports only one single format

### ISSUE THREAD

**Issue by jkumeboshi** [SimpleMobileTools/General-Discussion#132]
While Gallery2:Activity.fixDateTaken  function supports many EXIF date formats, the  SimpleCommons:ExifInterface.getExifDateTaken supports just one: "yyyy:MM:dd kk:mm:ss"
This prevents using the DateTaken attribute in different formats.

from fixDateTaken comments:
```
// some formats contain a "T" in the middle, some don't
// sample dates: 2015-07-26T14:55:23, 2018:09:05 15:09:05
```

**Comment by tibbi** [comment]
could you specify some other formats? Have you seen them being used somewhere?

**Comment by jkumeboshi** [comment]
Well, actually I just found the "2015-07-26T14:55:23" format on old pictures from my PC, edited using some EXIF tool, moved on my phone. I don't know any other format.
I discovered that since I saw that pictures from my PC didn't had the date taken shown in item's properties. Debugging then I discovered that getExifDateTaken didn't handle it, while fixDateTaken did (so, I guess, pictures are correctly ordered when fixDateTaken is used, also if the date is not shown in details).
